index.html

```diff
@@ -3581,7 +3581,8 @@ <h5>JWT Encoding</h5>
               </li>
               <li>
 <code>sub</code> MUST represent the <code>id</code> <a>property</a> contained
-in the <a>verifiable credential</a> <a>subject</a>.
+in the <code>credentialSubject</code>.
+Note. In the case of <code>bearer credential</code>s and <code>presentation</code>s, <code>sub</code> will be absent.
```

Comment:
```suggestion
<i>Note: In <code>bearer credentials</code> and <code>presentations</code>,
<code>sub</code> will not be present.</i>
```

Reply:
I would prefer normative statements here, not sure if thats possible.

Reply:
@David-Chadwick -- "Note. In" is not equivalent to "Note: In", even if you disagree with the `<i>` which sets the note apart.  (I think there's some CSS magic `<p class="note">` or similar that would be more appropriate, but I don't remember for sure what it is.) I would *VERY* much appreciate you not marking my comments or suggested changes as RESOLVED when they _**ABSOLUTELY ARE NOT!**_ 

"Resolution" requires either application of my suggested change, or conversation about it — which might include time in a WG concall — which arrives at a mutually acceptable "resolution", whether that is your original text, my original suggestion, or some other new suggestion springing from that conversation.

Reply:
I've now edited my requested change to use the CSS magic, which I found in another location.

Reply:
> I would prefer normative statements here, not sure if thats possible.

No, not possible at this time :( -- The Nov 9th v1.1 publication was the last time we could make normative changes. We can only make editorial changes to v1.1 from this moment forward.

Reply:
@David-Chadwick can you accept for reject this change suggestion?